File: backend/src/main/java/com/team01/project/domain/music/dto/SpotifyArtistResponse.java
package com.team01.project.domain.music.dto;

import java.util.List;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class SpotifyArtistResponse {
	private String id;
	private String name;
	private List<String> genres;
}

File: backend/src/main/java/com/team01/project/domain/music/dto/SpotifyArtistResponse.kt
package com.team01.project.domain.music.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class SpotifyArtistResponse(
    @JsonProperty("id")
    val id: String,

    @JsonProperty("name")
    val name: String,

    @JsonProperty("genres")
    val genres: List<String>? = null
)
